JBPM-5595: Upgrade lienzo-core to version 2.0.286-RELEASE.

romartin opened the pull request:
Hey @manstis @hasys 
And here are the changes for the `lienzo-core` version upgrade. Basically it's about removing the fixes that already present in the new `lienzo-core` release.
Depends on https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/416
Thanks!
Roger

romartin commented:
Jenkins please retest this.

manstis commented:
@etirelli Do you know of [DMN](https://kie-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/kie-wb-common-pullrequests/984/console) compilation failures?

manstis commented:
Jenkins please retest this.

manstis commented on kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-case-mgmt/kie-wb-common-stunner-case-mgmt-client/src/main/java/org/kie/workbench/common/stunner/cm/client/wires/CaseManagementDockingAndContainmentControlImpl.java:
```diff
@@ -22,15 +22,21 @@
 import com.ait.tooling.nativetools.client.collection.NFastArrayList;
-import org.kie.workbench.common.stunner.lienzo.wires.handlers.impl.WiresDockingAndContainmentControlImpl;
 
 public class CaseManagementDockingAndContainmentControlImpl extends WiresDockingAndContainmentControlImpl {
```
Sorry @romartin Your (I assume) conflict resolution of this class reverts it to the state it was at before https://github.com/kiegroup/kie-wb-common/pull/724

Can you please copy the code from [here](https://github.com/kiegroup/kie-wb-common/blob/master/kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-case-mgmt/kie-wb-common-stunner-case-mgmt-client/src/main/java/org/kie/workbench/common/stunner/cm/client/wires/CaseManagementDockingAndContainmentControlImpl.java) and correct any imports - you should not need to do anything else.

romartin replied:
Hey @manstis , I think this is correct now, consider that I have removed the custom control implementation from `lienzo-extensions`, so there is no `org.kie.workbench.common.stunner.lienzo.wires.handlers.impl` package now. You should be inheriting directly from the lienzo ones, isn't it?

etirelli commented:
@manstis I am now aware of any compilation failure with DMN. I just tried master again, and I don't see any failure.

romartin commented:
Hey @manstis , yeah sorry, I reverted some changes by error.. added a new commit, hope all good now, I have re-tested the showcase and it seems all fine.
Thanks!

manstis commented:
Jenkins please retest this.
